Appsilon/rhino
# Box function import count linter

### Changes
- Closes #525 
- Adds a `box_func_import_count_linter()`
- Adds the linter to the `.lintr` template
- Adds documentation

### How to test
1. Unit tests for `box_func_import_count_linter()` exist.
2. Locally install `rhino`, version should be `1.7.0.9003`, initialize a new rhino application then proceed to declare box imports.
3. `rhino::lint_r()` should find and point to all function imports that are more than 5 (default) per package/module.
4. Or, with an existing rhino app, add `box_trailing_commas_linter = rhino::box_func_import_count_linter()` to `.lintr`.
5. `rhino::lint_r()` should find and point to all function imports that are more than 5 (default) per package/module.
